eng/MSBuild/Packaging.targets

```diff
@@ -9,7 +9,7 @@
 
   <!-- packaging control -->
   <PropertyGroup>
-    <IsPackable Condition="'$(Stage)' == ''">false</IsPackable>
+    <IsPackable Condition="'$(Stage)' == '' and '$(IsPackable)' == ''">false</IsPackable>
```

Comment:
Same as above, I was trying to only change its value if it was previously unset, but I am unsure if this is the right choice.

Reply:
Probably needs a discussion with repo owners about how IsPackable/Stage interact.